### ISSUE TITLE
Strange worldgen issue with Thaumcraft?

### ISSUE THREAD

**Issue by ate0ate** [MinecraftModArchive/Dendrology#35]
Okay, here's a strange one, after updating to Ancient trees 1.5.1 I generated a new world and was exploring a thaumcraft magical forest when I encountered this...

http://imgur.com/a/ecu95

It appears that the usual mossy cobblestone boulders that generate in the magical forest biomes are somehow being generated using ancient trees planks. I have no idea what is going on beyond that. This is using the latest forge build for 1.7.10 (1456)

Also, oddly enough, the nearby BoP Highlands biome seemed to have the normal cobblestone boulders generate. Thanks!

**Comment by ate0ate** [comment]
After reverting back to Ancient Trees 1.4.0, where I did not notice this problem before, I found that there is still indeed a problem. Generating a fresh map with the same seed I found that the boulders are now invisible. For all intents and purposes there appears to be no blocks there, but the player cannot pass through the area nor interact with the invisible blocks in any way. They don't even appear in the NEI block name tooltip. Very odd indeed. Could it have something to do with meta data and Kiparis wood being designated as block ID 255?

http://i.imgur.com/tdE7vnc.png
